hermes-consumers/src/main/java/pl/allegro/tech/hermes/consumers/consumer/receiver/MessageReceiver.java

```diff
@@ -13,4 +16,10 @@
     default void stop() {}
 
     default void update(Subscription newSubscription) {}
+
```

Comment:
They are not required. That was just temporary change to compile code and run tests when multiple MessageReceiver implementation existed briefly. I forgot to clean this part up.